examples/window_run_return.rs

```diff
@@ -1,16 +1,7 @@
 #![allow(clippy::single_match)]
 
 // Limit this example to only compatible platforms.
-#[cfg(any(
-    target_os = "windows",
-    target_os = "macos",
-    target_os = "linux",
-    target_os = "dragonfly",
-    target_os = "freebsd",
-    target_os = "netbsd",
-    target_os = "openbsd",
-    target_os = "android",
-))]
+#[cfg(any(windows, macos, free_unix, android))]
```

Comment:
Would suggest to use `x11` and `wayland` here, since it's about display server, not the free_unix.